pkg/trace/info/stats.go

```diff
@@ -288,6 +284,49 @@ func (s *SpansMalformed) String() string {
 	return mapToString(s.tagValues())
 }
 
+const (
+	maxAbsSamplingPriority = 10
+	numPriorityCounters    = maxAbsSamplingPriority*2 + 1
+)
+
+// samplingPriorityStats holds the sampling priority metrics that will be reported every 10s by the agent.
+type samplingPriorityStats struct {
+	// tracesPerSamplingPriority holds counters for each priority in position MaxAbsPriorityValue + priority.
```

Comment:
```suggestion
	// tracesPerSamplingPriority holds counters for each priority in position maxAbsPriorityValue + priority.
	// Priority values are expected to be in the range [-10, 10].
```
I think this will save future readers some time with wrapping their head around things.

Reply:
Agreed, thanks!